README.yaml

```
@@ -20,65 +20,111 @@ copyrights:
 
 # List any resources helpful for someone to get started. For example, link to the hashicorp documentation or AWS documentation.
 references:
-  - name: "Terraform Standard Module Structure"
-    description: "HashiCorp's standard module structure is a file and directory layout we recommend for reusable modules distributed in separate repositories."
-    url: "https://www.terraform.io/docs/modules/index.html#standard-module-structure"
-  - name: "Terraform Module Requirements"
-    description: "HashiCorp's guidance on all the requirements for publishing a module. Meeting the requirements for publishing a module is extremely easy."
-    url: "https://www.terraform.io/docs/registry/modules/publish.html#requirements"
-  - name: "Terraform `random_integer` Resource"
-    description: "The resource random_integer generates random values from a given range, described by the min and max attributes of a given resource."
-    url: "https://registry.terraform.io/providers/hashicorp/random/latest/docs/resources/integer"
   - name: "Terraform Version Pinning"
     description: "The required_version setting can be used to constrain which versions of the Terraform CLI can be used with your configuration"
     url: "https://www.terraform.io/docs/configuration/terraform.html#specifying-a-required-terraform-version"
+  - name: "Terraform Cloud"
+    description: "Terraform Cloud is an application that helps teams use Terraform together."
+    url: "https://app.terraform.io/signup/account"
+  - name: "Terraform Enterprise/Cloud Documentation"
+    description: ""
+    url: "https://registry.terraform.io/providers/hashicorp/tfe/latest/docs"
 
 # Short description of this project
 description: |-
-  This is `terraform-example-module` project provides all the scaffolding for a typical well-built Cloud Posse module. It's a template repository you can
-  use when creating new repositories.
+  This is `terraform-tfe-cloud-infrastructure-automation` project enables the provisioning of workspaces & configuration in Terraform Cloud via YAML configuration.
+
+  NOTE: Requires [Terraform Cloud or Terraform Enterprise](https://www.hashicorp.com/products/terraform/pricing).
 
 # Introduction to the project
 #introduction: |-
 #  This is an introduction.
 
 # How to use this module. Should be an easy example to copy and paste.
 usage: |-
-  Here's how to invoke this example module in your projects
+  Here's how to invoke this example module in your projects:
 
   ```hcl
+  provider "tfe" {
+    version  = ">= 0.21.0"
+  }
+
   module "example" {
-    source = "https://github.com/cloudposse/terraform-example-module.git?ref=master"
-    example = "Hello world!"
+    source = "https://github.com/cloudposse/terraform-tfe-cloud-infrastructure-automation.git?ref=master"
+
+    config_file_path = "config"
+    organization     = var.organization
+
+    vcs_repo = {
+      branch             = "main"
+      ingress_submodules = true
+      oauth_token_id     = var.oauth_token_id
```

```suggestion
      # We recommend exporting the `FOOBAR` environment viable instead of passing a variable
      oauth_token_id     = var.oauth_token_id
```

(look up what `FOOBAR` should be)